lightstep/lightstep_b3_propagator.py

```diff
@@ -0,0 +1,127 @@
+from warnings import warn
+from re import compile as re_compile, IGNORECASE
+
+from opentracing.mocktracer.propagator import Propagator
+from opentracing.mocktracer.context import SpanContext
```

Comment:
We **cannot** use these components as they are a **mock** component, and as such, they are not meant to be used in production code. `SpanContext` needs to be used as an interface only, and in our internal code, we extend the `basictracer` package, so we internally end up using `basictracer.SpanContext`: https://github.com/opentracing/basictracer-python/blob/master/basictracer/context.py

Reply:
Fixed!